debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -53,6 +59,7 @@
     private static final String GET_LIST_OF_CDC_ENABLED_TABLES = "EXEC sys.sp_cdc_help_change_data_capture";
     private static final String GET_LIST_OF_NEW_CDC_ENABLED_TABLES = "SELECT * FROM cdc.change_tables WHERE start_lsn BETWEEN ? AND ?";
     private static final String GET_LIST_OF_KEY_COLUMNS = "SELECT * FROM cdc.index_columns WHERE object_id=?";
+    private static final String GET_SERVER_TIMEZONE = "DECLARE @TimeZone VARCHAR(50); EXEC MASTER.dbo.xp_regread 'HKEY_LOCAL_MACHINE','SYSTEM\\CurrentControlSet\\Control\\TimeZoneInformation','TimeZoneKeyName',@TimeZone OUT; SELECT @TimeZone";
```

Comment:
Would that work on Linux? If not (what I suppose to be the case), will this gracefully be handled?

Reply:
I just tried using mcr.microsoft.com/mssql/server:2017-CU8-ubuntu docker image : it works like a charm

Reply:
looks like wherever you have a Microsoft product, you have a registry ^_^